openfl-gramine/openfl.manifest.template

```diff
@@ -38,6 +38,10 @@ fs.mount.workspace.type = "chroot"
 fs.mount.workspace.path = "/workspace"
 fs.mount.workspace.uri = "file:/workspace"
 
+fs.mount.tmp.type = "tmpfs"
+fs.mount.tmp.path = "/tmp"
+fs.mount.tmp.uri = "file:/tmp"
```

Comment:
the "uri" attribute is not relevant when using tmpfs, so this line can be removed

Reply:
Done!